### DISCUSSION TITLE
期望higress的插件配置交互与商业版一致

### DISCUSSION THREAD

**Discussion by wkeylin** [higress-group/higress#1274] [Ideas]
## Why do you need it?
目前插件配置支持实例级，路由级，域名级，商业版的这些级别配置都在统一的入口，但是开源版本是分别在三处，并且如果是一个插件作用于n个路由，则需要在这n个路由配置都是一样的配置

## How could it be?
如果现有接口支持，我可以支持前端改动

## Other related information
<img width="1456" alt="image" src="https://github.com/user-attachments/assets/345806ee-4dbd-4abc-8c1e-5bd490607f12">
<img width="1511" alt="image" src="https://github.com/user-attachments/assets/519842e6-5d5b-49ff-a7f7-774f78273dfc">

**Comment by johnlanni** [comment]
@wkeylin 欢迎参与控制台贡献哈。可以加这个钉钉群：https://qr.dingtalk.com/action/joingroup?code=v1,k1,JkUnEbivc8aV8nccAOK/6dwYL/tgzNDeAL3hTJ3pyZc=&_dt_no_comment=1&origin=11
